oeps/processes/oep-0021-proc-deprecation.rst

```diff
@@ -275,15 +277,47 @@ Do the following to document your proposal:
       **Deprecation** section. See Deprecated_ for considerations.
    #. If automated migration will be needed, explain your migration plan in the
       **Migration** section.
-   #. If there is any additional publicly shareable information or data from
-      your earlier analysis, include that in the **Additional Info** section.
+   #. **Additional Info**:
+
+      - If there is any additional publicly shareable information or data from
+        your earlier analysis, include that in the **Additional Info** section.
+      - If you know who plans to take ownership of the ticket, include the
+        name in the **Additional Info** section. **Important**: the deprecation
+        ticket will not move forward until someone takes ownership.
 
 #. Check that your ticket appears on the `DEPR Project Board`_.
 
 .. note::
   You can create a ticket even if you don't yet have all the information
-  gathered yet. You simply must wait until you have completed the issue before
-  moving on to communicating.
+  gathered yet. You simply must wait until you have completed the issue and
+  have an owner before moving on to the formal **Communication** step.
+
+(Optional) Start a discussion
```

Comment:
So, a bit lower in this document there is a section that says to create an issue on the forums, and it's _not_ optional. I think you should add in this language about seeking an owner to that section.

https://docs.openedx.org/projects/openedx-proposals/en/latest/processes/oep-0021-proc-deprecation.html#to-the-open-edx-discourse

Reply:
It's a different phase (the non-optional phase is once you have an owner; the optional phase is when you're just noodling around). Let me think about how to clarify that.

Reply:
I think the modifications I made to this added some clarity, but I'm still not thrilled with what I wrote.

Reply:
I think it's good. One thing you could consider doing is adding a link to the `Announce` section, maybe something like: ```Note: This does not replace the formal `announcement <link_to_announce_ref>_` of the DEPR ticket```. But otherwise I think it's good.